### PR TITLE
Project: Network 6 - DLNA Code simplification and standardisation.

### PR DESCRIPTION
**Pull request by BaronGreenback**
**Looks like a large change - but it isn't, it just touches many usings.**

Main change is the removal of a level of coding abstraction in dlnamanager.cs and deviceinfo.cs - which simplifies the code.
(There was no need to create a new DeviceIndentification object, just for comparison purposes.)

Moved IDlnaManager.cs into the Emby.Dlna namespace to permit this change. (couldn't see DeviceInfo.cs)

Renamed DeviceInfo.cs to PlayToDeviceInfo to remove duplication with code in MediaBrowser.Model/Devices

Moved the following file into the MediaBrowser.common.dlna namespace (where the other dlna classes are stored.) 

 - PlayToDeviceInfo.cs, 
 - Argument.cs, 
 - DeviceIcon.cs, 
 - DeviceService.cs, 
 - ServiceAction.cs 
 - StateVariables.cs

The other changes are due namespace changes and nullables.

### CODE REVIEW

File: Emby.Dlna/PlayTo/DeviceInfo.cs [hunk 1]
```diff
@@ -1,64 +0,0 @@
-#pragma warning disable CS1591
```
Comment (BaronGreenback) [inline Emby.Dlna/PlayTo/DeviceInfo.cs:1]: Renamed DeviceInfo.cs file with ToDeviceIndentification() removed as it's no longer required.

File: MediaBrowser.Model/Dlna/PlayToDeviceInfo.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#pragma warning disable CS1591
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Dlna/PlayToDeviceInfo.cs:1]: DeviceInfo.cs file with ToDeviceIndentification() removed as it's no longer required.

File: Emby.Dlna/Common/Argument.cs [hunk 3]
```diff
@@ -1,13 +0,0 @@
-#pragma warning disable CS1591
```
Comment (BaronGreenback) [inline Emby.Dlna/Common/Argument.cs:1]: Moved file

File: MediaBrowser.Model/Dlna/DeviceService.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#pragma warning disable CS1591
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Dlna/DeviceService.cs:1]: Moved from emby,dlna and made nullable.

File: MediaBrowser.Model/Dlna/DeviceService.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace MediaBrowser.Model.Dlna
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Dlna/DeviceService.cs:3]: Moved from emby,dlna and made nullable.

File: MediaBrowser.Model/Dlna/DeviceService.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public class DeviceService
+    {
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Dlna/DeviceService.cs:6]: Moved from emby,dlna and made nullable.

File: MediaBrowser.Model/Dlna/PlayToDeviceInfo.cs [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Dlna/PlayToDeviceInfo.cs:2]: Moved from emby,dlna and made nullable.

File: MediaBrowser.Model/Dlna/Argument.cs [hunk 8]
```diff
@@ -0,0 +1,13 @@
+#pragma warning disable CS1591
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Dlna/Argument.cs:1]: Moved file and made nullable.

**Comment by crobibero** [comment]
Can you also remove the `CS1591` warnings? 
Since you spent so much time in there it's probably best for you to describe what each property does

**Comment by BaronGreenback** [comment]
Have commented the model\dlna code attributes. Lots of them are just standard.
Will be updating the other in the next PR.

**Review by crobibero — APPROVED** [review]

**Pull request closed without merging**